examples/chem-sync-local-flask/ruff.toml

```diff
@@ -4,7 +4,7 @@ target-version = "py311"
 line-length = 110
 
 select = ["ALL"]
-ignore = ["D100", "D101", "D103", "D104", "EM101", "EM102", "S101", "TRY003"]
+ignore = ["D100", "D101", "D103", "D104", "EM101", "EM102", "S101", "TRY003", "TRY301"]
```

Comment:
Added `TRY301` to ignore: `local_app/benchling_app/handler.py:36:13: TRY301 Abstract `raise` to an inner function`